rwiki/rwiki-tool/tool/src/bundle/uk/ac/cam/caret/sakai/rwiki/tool/bundle/Messages_eu.properties

```diff
@@ -515,3 +515,19 @@ save.noupdate_permission=Ez duzu baimenik orrialde hau eguneratzeko.
 
 updatepermissioncmd.content_changed=Edukia aldatu da azkenekoz ikusi zenuenetik. Eguneratu, arren, eduki berria edo gainidatzi bialitako edukia.
 updatepermissioncmd.noupdate_permission=Ez duzu baimenik orrialde hau eguneratzeko.
+
+### S2U-39 5.2.3.1 Wiki: Add group support to Wiki pages
+save.content_no_groups = Talde bat aukeratu behar duzu, gutxienez.
+
+view_groups_text       = Orrialde hau taldeentzat bakarrik dago erabilgarri
+
+displayto_access          = Sarbidea
+displayto_access_subtitle = Aukeratu nork ikus dezakeen orrialde hau
+displayto_access_all      = Erakutsi orrialde hau erabiltzaile guztiei
+displayto_access_groups   = Erakutsi orri hau hautatutako taldeei soilik
+
+groups_multiple_all         = Hautatu talde guztiak
+groups_multiple_nonselected = Gutxienez talde bat hautatu behar duzu
+groups_multiple_allselected = Hautatutako talde guztiak
+groups_multiple_nselected   = hautatutako taldeak
+groups_multiple_filtertext  = Bilaketa taldea
```

Comment:
@pablosanroman could you please provide the translations for this?

Reply:
Hi @mpellicer , where is the original in english?I guess I have to check these translations but I need the original.

Reply:
Hi @mpellicer , I'm not very handy here in github, I'm sorry. Here are the translations: 


save.content_no_groups = Talde bat aukeratu behar duzu, gutxienez.

view_groups_text       = Orrialde hau taldeentzat baino ez dago eskuragarri:

displayto_access          = Sarbidea
displayto_access_subtitle = Aukeratu nork ikus dezakeen orrialde hau
displayto_access_all      = Erabiltzaile guztiek
displayto_access_groups   = Hautatutako taldekoek

groups_multiple_all         = Dena hautatu
groups_multiple_nonselected = Gutxienez talde bat hautatu behar duzu
groups_multiple_allselected = Hautatutako guztiak
groups_multiple_nselected   = hautatutakoak
groups_multiple_filtertext  = Bilatu taldea

Reply:
No problem! Thanks a ton

Reply:
@PabloSanRoman You've done it more than perfect, thanks!